content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+This will create a subfolder in themes/my-hugo-theme that contains the files for your theme. 
 
----
+#### Method 3: Install the theme as Hugo module
 
-## Working with source files
+If the theme is available as a Hugo module you can install it as module. Hugo will then download the theme when it compiles or serves your website.
 
-Now that you've got your theme running and understand the compiling process, let's discuss the proper ways to edit and customize your theme. You can directly customize your template from the source file. For a better experience, open the whole theme folder with a text-editor or IDM (i.e. [Visual Studio Code](https://code.visualstudio.com/)).
+Initialize the use of modules in your website with the following command:
 
-### Site configuration
+```bash
+hugo mod init github.com/my-username/my-hugo-theme
+```
 
-You have a `config.toml` file in the `exampleSite` folder. Which gives you the ability to configure your site, such as `changing logo` or `customize menu`.
+After that you will find a file `go.mod` in your root directory. Add this file to your repository.
 
-#### Default configuration
+Your website is now ready to receive modules. Set up the theme as module in your configuration:
 
-Default configuration is those which come with the Hugo library. Remember that, all configuration does not come with every theme.
+```toml
+[[module]]
+[[module.imports]]
+path = "github.com/themeauthor/themename"
+```
 
- * `baseURL` defines your site URL. Give your site URL here (i.e. https://examplesite.org).
+{{% notice warning %}}Do this change in your config.toml file ***after*** you copy the exampleSite into your root directory (see the next step). {{% / notice %}}
 
- * `title` is for your site title. Which shown on the browser tab.
+The next time you run Hugo it will download the theme. You will not see that theme in your `themes` folder, because Hugo caches it in a temporary directory. But no need to worry, it will find the theme files.
 
- * `theme` name is already provided with every theme. don't need to customize this theme name.
+You will find a file called `go.sum` in your root directory. Add this file to your repository.
 
- * `summaryLeangth` is for the post excerpt limit. you can set your post excerpt length as you want to show.
+### Step 5: Copy the sample site
 
- * `paginate` set your pagination for page items. Such as blog posts, portfolio items, etc. If you set `paginate = 10` then it will show 10 posts on a page.
+Every theme comes with a folder named `exampleSite`. Copy the folders and files inside of that folder into your root directory. Be careful not to overwrite anything you already changed, but expect to override some files and folders that were created by step 2 above.
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)